packages/markup-include/src/data/markup-include-example.json

```diff
@@ -0,0 +1,362 @@
+[
```

Comment:
I don't think we need any addititional data sources. we should be able to use the ones we've already stored